packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -258,6 +258,7 @@ export class FASTCombobox extends FormAssociatedCombobox {
             ) as FASTListboxOption | null;
 
             if (!captured || captured.disabled) {
+                this.open = false;
```

Comment:
Adding this here causes a few issues:

- Clicking the input toggles the `open` state. This primarily shows up when highlighting text in the input field. For instance, double-clicking to highlight a word or triple-clicking to highlight the entire input now opens and closes the listbox with each click. Similarly, clicking and holding the mouse button to highlight contents of the input toggles the `open` state when the mouse button is released.
- Clicking a disabled option closes the listbox.

Reply:
@zpeterson : has this change been marked as fixed?